sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Release History
 
 ## 7.3.0-beta.3 (Unreleased)
+### New Features
+- Added support receiving dead-letter queue (DLQ) for 'ServiceBusProcessorClient' and session receiver using 'subQueue'
```

Comment:
"Added support of receiving dead-letter queue (DLQ) to 'ServiceBusProcessorClient' by using 'subQueue'". "session receiver" may mean something else but not the processor client.